packages/block-library/src/cover/edit.native.js

```diff
@@ -68,6 +69,31 @@ const INNER_BLOCKS_TEMPLATE = [
 const COVER_MAX_HEIGHT = 1000;
 const COVER_DEFAULT_HEIGHT = 300;
 
+const COVER_DEFAULT_PALETTE = {
```

Comment:
To avoid duplicating the colors again, we could import them from the default settings and here we can just filter that list with the colors we want for the block, so I'd suggest importing the colors from:

`const { getSettings } = select( 'core/block-editor' );`

And then filter the colors to return the values of the ones we want, what do you think?